src/backend/matrix_client.py

```diff
@@ -1894,6 +1905,9 @@ async def register_nio_room(
             local_highlights = local_highlights,
 
             lexical_sorting = self.backend.ui_settings["lexicalRoomSorting"],
+            bookmarked = room.room_id in
+                self.backend.ui_settings["roomBookmarkIDs"].get(self.user_id,
+                                                                {}),
```

Comment:
```suggestion
            bookmarked = 
                room.room_id in self.backend.ui_settings["roomBookmarkIDs"]
                                            .get(self.user_id, {}),
```
Looks a bit more natural IMO

Reply:
I refactored it a bit different, by putting bookmarks in a variable